File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        for(int i = 0; i< numbers.length;i++){
            for(int j = numbers.length - 1; j>i; j--){
                if(numbers[i] + numbers[j] == target) {
                    int[] answer = new int[2];
                    answer[0] = i + 1;
                    answer[1] = j + 1;
                    return answer;
                }
            }
        }
        return new int[2];
    }
}

File: Data Structures & Algorithms/search-insert-position/submission-0.py
class Solution:
    def searchInsert(self, nums: List[int], target: int) -> int:

        lo = 0
        hi = len(nums) - 1
        while lo < hi:
            mid = (lo + hi)//2
            if nums[mid]<target:
                lo = mid+1
            else:
                hi = mid
        if nums[lo] >= target:
            return lo
        else:
            return lo + 1
            
        